lizmap/modules/view/controllers/lizMap.classic.php

```diff
@@ -128,7 +128,10 @@ public function index()
 
         // the html response
         $rep = $this->getResponse('htmlmap');
-        $rep->addJSLink(jUrl::get('view~translate:index'));
+        // Get Lizmap version from project.xml
+        $xmlLoad = simplexml_load_file(jApp::appPath('project.xml'));
```

Comment:
I don't like it. opening an xml file is slooooww. we should find an other value....

Reply:
I don't know where else to find this value. It might be slow but it is slow only the first time.

Reply:
ok